README.md

```diff
@@ -54,6 +54,27 @@ You will be prompted to supply your trello api key and token
 
 Navigate to port 5000 on your host to view the website
 
+## Running the app in dev mode with Docker
+
+Make sure you have Docker Desktop installed
+
+Populate the environment variables in a `.env` file like usual
+
+Run `docker build --target development --tag todo-app:dev` to build the image
+
+Run `docker run -d -p 8000:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app --env-file=.env todo-app:dev` to run the container in dev mode with hot reloading
+
+Navigate to `localhost:8000` to open the website
+
+## Running the app in production mode with Docker Compose
+
+Make sure you have Docker Desktop installed
+
+Populate the environment variables in a `.env` file like usual
+
+Run `docker compose up` from in root directory
```

Comment:
Nice 👍 
Bonus note - I often find myself falling foul of compose not rebuilding the image after I've updated the code/Dockerfile, so you may also find it useful to have a default `--build` argument on this

Bonus - you may find an explicit command for building the production image is still helpful even if docker compose usually handles it for you, since eventually we'll want to do things with that image (push it to a repo, pull it to the service that will deploy it etc.)